Cargo.toml

```diff
@@ -95,6 +95,7 @@ members = [
     "examples/txpool-tracing/",
     "testing/ef-tests/",
     "testing/testing-utils",
+    "crates/db-common",
```

Comment:
this should be moved into the crates/storage folder

also please put the new entry in the alphabetically correct position.